app/src/main/res/layout/vungle_native_ad.xml

```diff
@@ -0,0 +1,103 @@
+<com.vungle.warren.NativeAdLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/native_ad_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```

Comment:
Can you rewrite this using a ConstraintLayout? I know this was probably copied from the Vungle docs but nested LinearLayouts with weights is considered an anti pattern now.